inst/sql/dwc_occurrence.sql

```diff
@@ -0,0 +1,252 @@
+  CASE
+    WHEN TRIM(LOWER(animal.sex)) IN ('male', 'm') THEN 'male'
+    WHEN TRIM(LOWER(animal.sex)) IN ('female', 'f') THEN 'female'
+    WHEN TRIM(LOWER(animal.sex)) IN ('hermaphrodite') THEN 'hermaphrodite'
+    WHEN TRIM(LOWER(animal.sex)) IN ('unknown', 'u') THEN 'unknown'
+    -- Exclude transitional, na, ...
+  END                                   AS sex,
+  CASE
+    WHEN event.protocol = 'release' THEN -- Only at release, can change over time
+      CASE
+        -- Follows http://vocab.nerc.ac.uk/collection/S11/current/, see https://github.com/inbo/etn/issues/262
+        WHEN TRIM(LOWER(animal.life_stage)) IN ('juvenile', 'i', 'fii', 'fiii') THEN 'juvenile'
+        WHEN TRIM(LOWER(animal.life_stage)) IN ('sub-adult', 'fiv', 'fv', 'mii', 'silver') THEN 'sub-adult'
+        WHEN TRIM(LOWER(animal.life_stage)) IN ('adult', 'mature') THEN 'adult'
+        WHEN TRIM(LOWER(animal.life_stage)) IN ('immature', 'imature') THEN 'immature'
+        WHEN TRIM(LOWER(animal.life_stage)) IN ('smolt') THEN 'smolt'
+        -- Exclude unknown, and other values
+      END
+  END                                   AS lifeStage,
+  'present'                             AS occurrenceStatus,
+  animal.id_pk                          AS organismID,
+  animal.animal_nickname                AS organismName,
+-- EVENT
+  animal.id_pk || '_' || tag_device.serial_number || '_' || event.protocol AS EventID,
+  animal.id_pk || '_' || tag_device.serial_number AS parentEventID,
+  TO_CHAR(event.date, 'YYYY-MM-DD"T"HH24:MI:SS"Z"') AS eventDate,
+  event.protocol                        AS samplingProtocol,
+  CASE
+    WHEN event.protocol = 'capture' THEN
+      'Caugth using ' || TRIM(LOWER(animal.capture_method))
+    WHEN event.protocol = 'release' THEN
+      manufacturer.project || ' ' || tag_device.model || ' tag ' ||
+      CASE
+        WHEN LOWER(animal.implant_type) = 'internal' THEN 'implanted in '
+        WHEN LOWER(animal.implant_type) = 'external' THEN 'attached to '
+        ELSE 'implanted in or attached to ' -- Includes `Acoutic and pit`, ...
+      END ||
+      CASE
+        WHEN TRIM(LOWER(animal.wild_or_hatchery)) IN ('wild', 'w') THEN 'free-ranging animal'
+        WHEN TRIM(LOWER(animal.wild_or_hatchery)) IN ('hatchery', 'h') THEN 'hatched animal'
+        ELSE 'likely free-ranging animal'
+      END
+  END                                   AS eventRemarks,
+-- LOCATION
+  NULL                                  AS locationID,
+  event.locality                        AS locality,
+  event.latitude                        AS decimalLatitude,
+  event.longitude                       AS decimalLongitude,
+  CASE
+    WHEN event.latitude IS NOT NULL THEN 'EPSG:4326'
+  END                                   AS geodeticDatum,
+  CASE
+     -- Assume coordinate precision of 0.001 degree (157m) and recording by GPS (30m)
+    WHEN event.latitude IS NOT NULL THEN 187
```

Comment:
`WHEN event.latitude IS NOT NULL THEN 187` should be 157 as per comment on line 177

Reply:
157 for rounding + 30 for GPS